MatrixSDK/Utils/Profiling/MXTaskProfile.h

```diff
@@ -18,13 +18,28 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef NSString *const MXTaskProfileName NS_TYPED_EXTENSIBLE_ENUM;
```

Comment:
For readability and as `MXTaskProfileName` can grow, I think it worth to create an `MXTaskProfileName.h` and `.m`